docs/docs/guides/integrations/index.md

```diff
@@ -5,3 +5,4 @@ The Weave ecosystem contains automatic logging integrations for popular AI libra
 - [openai](/guides/integrations/openai): the `openai` library.
 - [anthropic](/guides/integrations/anthropic): the `anthropic-sdk-python` library.
 - [mistralai](/guides/integrations/mistral): the `mistralai` library.
+- [llamaIndex](/guides/integrations/llamaindex): the `llamaIndex` library.
```

Comment:
I think we should be capitalizing these correctly, i.e. `LlamaIndex` if we are referring to the framework, `llama-index` if we are referring to the Python package.